src/Explorer/Notebook/NotebookUtil.ts

```diff
@@ -102,25 +102,19 @@ export class NotebookUtil {
   }
 
   public static findFirstCodeCellWithDisplay(notebookObject: ImmutableNotebook): number {
-    let codeCellCount = -1;
+    let codeCellCount = 0;
```

Comment:
Add a comment as to why we're now skipping the first cell. Btw, since the newly injected cell isn't a code cell, I would keep starting from -1. It won't hurt and we're not adding more dependency to this special cell. If later on, the mechanism changes we won't have to go back here and fix.

Reply:
It's really an index rather than a count. Rename to `cellIndex`?

Reply:
this change was independent of the linkInjection change. I reworked the method to use base index as 0, just to make it look better.will rename to codecellIndex